bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/internal/config/ClangdPreferredOptions.java

```diff
@@ -72,7 +72,7 @@ public List<String> additionalOptions() {
 		if (options.isBlank()) {
 			return new ArrayList<>();
 		}
-		return Arrays.asList(options.split("\\s+")); //$NON-NLS-1$
+		return Arrays.asList(options.split(System.lineSeparator()));
```

Comment:
Minor observation is that on Windows this won't split `\n` separated options anymore. Should this be split on `"\r?\n"` (not sure if escaping is correct, but I hope you understand my meaning?)

Reply:
You're right. The originally implementation used `\R` in the regex. This might be the better implementation.